subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/NotificationService.kt

```diff
@@ -54,19 +54,21 @@ class NotificationService(
             subscriptionId = subscription.id,
             data = buildNotifData(entity, subscription.notification)
         )
-
-        if (subscription.notification.endpoint.uri.toString() == "embedded-firebase") {
+        val uri = subscription.notification.endpoint.uri.toString()
+        logger.info("Notification is about to be sent to $uri")
```

Comment:
I'm wondering if you could log the success case rather than this one.

Reply:
Yes, you're right, I'll add a log for success